client/layout/index.jsx

```diff
@@ -178,13 +179,6 @@ class Layout extends Component {
 		};
 
 		const { shouldShowAppBanner } = this.props;
-
-		if ( isAutomatticTeamMember( this.props.teams ) ) {
-			config.enable( 'nav-unification' );
-		} else {
-			config.disable( 'nav-unification' );
-		}
-
```

Comment:
We cannot rely on the feature-flag. We want this feature to be enabled also in production for a12s. This is very useful for testing from mobile devices or also in every-day use which in turn increases the chances to find bugs.

Reply:
As invasive as it feels at first glance, this makes sense for such a huge-impact change. Thanks for clarifying, that sounds good to me 👍